src/App.js

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import './App.css';
-import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import CalculatorPage from './components/calculator-page';
+import QuotesPage from './components/quotes-page';
+import HomePage from './components/home-page';
+import Navbar from './components/Navbar';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
-      </div>
-    );
-  }
-}
+const App = () => (
+  <Router>
+    <div className="App">
+      <Navbar />
+      <Routes>
+        <Route path="math-magicians/calculator" element={<CalculatorPage />} />
+        <Route path="math-magicians/quotes" element={<QuotesPage />} />
+        <Route path="math-magicians/" element={<HomePage />} />
+      </Routes>
```

Comment:
- [ ] Nice use of React `Route` here 👍 
- [ ] **[OPTIONAL]** To improve the UI of your app, it would be nice if you set up routes so that root path `"/"` goes to HomePage, `"/calculator"` goes to CalculatorPage, and `"/quotes"` goes to QuotesPage instead of `"math-magicians/"`,  `"math-magicians/calculator"`, and `"math-magicians/quotes"` respectively. You would also need to make the changes in your `Navbar.js` Link elements too.